src/bpmncwpverify/core/expr.py

```diff
@@ -64,8 +63,7 @@ def _parse_expressions(parser: ExprParser) -> Result[ExprParser.StartContext, Er
         return Success(tree)
     except ParseCancellationException as exception:
         msg = str(exception)
-        failure_value = ExceptionError(msg)
-        return Failure(failure_value)
+        return Failure(msg)
```

Comment:
OK. This line should also not type check since `str` is _not_ a subtype of `Error`? Is your type checking turned on in your editor? I'm concerned that these errors are not showing up in `pre-commit` or on your editor with a `mypy` plugin.  It makes me wonder if maybe the type-checking is not working again.